diff --git a/node_modules/tesseract.js/src/createWorker.js b/node_modules/tesseract.js/src/createWorker.js
index 89b53b8..0aa0039 100644
--- a/node_modules/tesseract.js/src/createWorker.js
+++ b/node_modules/tesseract.js/src/createWorker.js
@@ -145,7 +145,7 @@ module.exports = async (langs = 'eng', oem = OEM.LSTM_ONLY, _options = {}, confi
     // and then uses `worker.reinitialize` to switch to the Legacy engine.
     // However, the correct data will still be downloaded after initialization fails
     // and this can be avoided entirely if the user loads the correct data ahead of time.
-    const langsArr = typeof langs === 'string' ? langs.split('+') : langs;
+    const langsArr = Array.isArray(langs) ? langs : (typeof langs === 'string' ? langs.split('+') : []);
     const _langs = langsArr.filter((x) => !currentLangs.includes(x));
     currentLangs.push(..._langs);
 
